prometheus-rules/prometheus-openstack-rules/aggregations/haproxy.rules

```diff
@@ -0,0 +1,26 @@
+groups:
+- name: haproxy.rules
+  rules:
+  - record: global:api_errors_per_request_sli:ratio_rate5m
```

Comment:
Why don't we take the configurable time here from the previous chart? Even though we might not need the flexibility, it would avoid the duplication and future errors in case of changes needed. (same for ingress rules above)

https://github.com/sapcc/helm-charts/blob/49dc3518b8cc0b501bb35f03b45de47302f5b48d/openstack/sre/templates/prometheus-aggregations.yaml#L33-L40